Games/TinyBlocksGS/TinyBlocksGS.py

```diff
@@ -8,13 +8,22 @@
 import random
 import machine
 
+import thumbyGrayscale
+thumby.display = thumbyGrayscale.display
+thumbyGrayscale.display.startGPU()
+
 machine.freq(48000000)
 
 TinyBlocksSplash = bytearray([170,85,170,85,170,85,170,85,170,85,170,85,170,21,10,5,2,1,0,1,0,1,0,1,0,1,0,1,0,1,8,121,8,1,120,1,120,16,33,120,1,24,97,24,1,0,1,0,1,0,1,0,1,0,1,2,5,10,85,170,85,170,85,170,85,170,85,170,85,170,85,170,
-           170,85,170,85,170,85,170,85,170,85,170,85,2,0,0,0,0,0,254,34,34,34,34,220,0,254,0,0,0,0,252,6,2,2,6,252,0,252,6,2,2,2,0,254,16,104,132,2,0,28,34,34,34,194,0,0,0,0,0,2,85,170,85,170,85,170,85,170,85,170,85,170,
-           170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,7,4,4,4,4,3,0,7,4,4,4,0,3,6,4,4,6,3,0,3,6,4,4,4,0,7,0,0,1,6,0,4,4,4,4,3,0,0,0,0,0,0,85,170,85,170,85,170,85,170,85,170,85,170,
-           170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,85,170,85,170,85,170,85,170,85,170,85,170,
-           170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,85,170,85,170,85,170,85,170,85,170,85,170])
+                    170,85,170,85,170,85,170,85,170,85,170,85,2,0,0,0,0,0,254,34,34,34,34,220,0,254,0,0,0,0,252,6,2,2,6,124,0,188,6,2,2,2,0,254,16,104,128,2,0,28,34,34,34,194,0,0,0,0,0,2,85,170,84,170,84,170,84,170,84,170,85,168,
+                    170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,7,4,4,4,4,3,0,7,4,4,4,0,3,6,4,4,6,3,0,3,6,4,4,4,0,7,0,0,1,6,0,4,4,4,4,1,0,0,0,0,0,0,21,42,21,170,85,170,85,170,21,138,85,162,
+                    170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,190,194,1,33,33,240,49,8,21,20,21,253,252,253,253,248,250,252,120,240,128,0,0,0,0,0,0,69,162,80,162,81,168,85,170,85,170,85,170,
+                    170,85,170,85,170,85,170,85,170,85,170,85,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,15,16,32,33,2,34,47,52,36,100,132,166,186,63,159,191,159,207,107,60,7,7,6,0,0,0,0,0,85,170,85,170,85,170,85,170,85,170,85,170])
+TinyBlocksSplashSHD = bytearray([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```

Comment:
If this was copied straight from the dropIn branches, then I just want to note that this Sprite had a fairly ugly test scribble over the top of it. I was being lazy and was just testing it. I think its fine to leave it in, but just a note that it could benefit from some improved artwork, if it ever gets pushed to the Arcade.

Reply:
Alright, thanks for the heads up 😄